dashboard/app/views/levels/_unplug.html.haml

```diff
@@ -1,5 +1,5 @@
 - unplugged = @level.type == 'Unplugged' ? @level.name : @game.name
-- video_key = try_t("data.unplugged.#{unplugged}.video")
+- video_key = @level.video_key || try_t("data.unplugged.#{unplugged}.video")
```

Comment:
The original `data.unplugged...video` key is our alternate (legacy?) process for defining 'unplugged' videos, correct? Is this i18n-driven process being deprecated as part of this change in favor of the more common video-selection process used by the other level types?